### PR TITLE
remove default swmm path

### PR DESCRIPTION
**Pull request by tproduit**
To solve https://github.com/QGEP/QGEP/issues/543

### CODE REVIEW

File: qgepplugin/processing_provider/provider.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def load(self):
         ProcessingConfig.addSetting(Setting(self.name(),
                                             'SWMM_PATH',
                                             self.tr("SWMM executable"),
```
Comment (m-kuhn) [inline qgepplugin/processing_provider/provider.py]:
```suggestion
                                            self.tr(r'SWMM executable (e.g. C:\Program Files (x86)\EPA SWMM 5.1.013\swmm55.exe)'),
```

How about this to help people with the configuration?

Comment (tproduit) [reply qgepplugin/processing_provider/provider.py]: Seems clever

**Comment by m-kuhn** [comment]
@pep8speaks pep8ify

**Comment by sjib** [comment]
Looks good to me

**Pull request merged**